ldap_auth_provider.py

```diff
@@ -50,7 +50,7 @@ class LDAPMode(object):
 
 
 class LdapAuthProvider(object):
-    __version__ = "0.1"
+    __version__ = "0.2"
```

Comment:
I thought since we were adding a relatively big feature, and we should get into the habit of bumping the version number when we do that.

Maybe that doesn't make sense though for this code.

Reply:
I think if we do a release that should be done separately from this PR tbh.

Reply:
Ah ok, good point.

Reply:
Also not sure why we have this line and this:

https://github.com/matrix-org/matrix-synapse-ldap3/pull/58/files/3c0b342cd0a89b33ba981b52e3178c39a2bc98d0#diff-684dbcb9cbdc9e9506d15d554744f81aR25

Reply:
Heh, no idea